addons/purchase_stock/tests/test_replenish_wizard.py

```diff
@@ -35,6 +35,15 @@ def setUpClass(cls):
         cls.uom_pack_6 = cls.env.ref('uom.product_uom_pack_6')
         cls.wh = cls.env['stock.warehouse'].search([('company_id', '=', cls.env.user.id)], limit=1)
 
+    def _get_purchase_order_from_replenishment(self, replenish_wizard):
+        picking = replenish_wizard.launch_replenishment()
+        links = picking.get("params", {}).get("links")
```

Comment:
It still not a picking
```suggestion
        notification = replenish_wizard.launch_replenishment()
        links = notification.get("params", {}).get("links")
```